packages/wdio-appium-service/src/utils/getFilePath.js

```diff
@@ -0,0 +1,21 @@
+import path from 'path'
```

Comment:
Same here, you can use 'import {join, resolve} from "path"'

Reply:
Fixed 👍